### ISSUE TITLE
/Overview.htm missing

### ISSUE THREAD

**Issue by irwinwilliams** [dotnet/BenchmarkDotNet#2129]
Hey, so I googled my way to finding the repo, and the top result for me was this:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/1428703/193609068-48e49a3b-8526-44b4-90cc-0c5dda7d83ad.png">
ALT: A google link taking you to https://benchmarkdotnet.org/Overview.htm 

But that results in a 404.

The search result might be out of your control, (maybe a site admin could login to google sitemaster and address it), but perhaps in the meantime, adding back an /Overview.htm that redirects to home might help. At first I thought, wait, did I just stumble upon a dead site??

But then I saw the address bar and backspaced my way out of doom. 

Cheers.

**Comment by AndreyAkinshin** [comment]
@irwinwilliams, thanks for the report, it's fixed now. It's quite strange that Google still provides a link to this page because it just redirects you to another page. Recently, I have set up an automatic workflow for documentation deployment and skipped redirect pages since they are obsolete during last several years. Currently, I just manually uploaded the redirect pages, but I will try to find a better permanent solution.
